### PR TITLE
add Wang V2 keyboard

### PR DESCRIPTION
**Pull request by oceeean**
## Description

add Wang V2, 40% ortholinear keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/ocean/wang_v2/wang_v2.h [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C, \
+  K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, \
+  K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B, \
+  K30, K31, K32, K33,      K35, K36,      K38, K39, K3A, K3B \
+) \
+{ \
+  { K00, K01, K02, K03, K04,   K05,   K06, K07,   K08, K09, K0A, K0B, K0C }, \
+  { K10, K11, K12, K13, K14,   K15,   K16, K17,   K18, K19, K1A, K1B }, \
+  { K20, K21, K22, K23, K24,   K25,   K26, K27,   K28, K29, K2A, K2B }, \
+  { K30, K31, K32, K33, KC_NO, K35,   K36, KC_NO, K38, K39, K3A, K3B }, \
```
Comment (fauxpark) [inline keyboards/ocean/wang_v2/wang_v2.h:30]:
```suggestion
  { K10, K11, K12, K13, K14,   K15,   K16, K17,   K18, K19, K1A, K1B, KC_NO }, \
  { K20, K21, K22, K23, K24,   K25,   K26, K27,   K28, K29, K2A, K2B, KC_NO }, \
  { K30, K31, K32, K33, KC_NO, K35,   K36, KC_NO, K38, K39, K3A, K3B, KC_NO }, \
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/ocean/wang_v2/readme.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Wang V2
+
+![Wang V2](https://media.discordapp.net/attachments/406032987243806721/859276477354868736/20210629_112245.jpg?width=1166&height=657)
```
Comment (fauxpark) [inline keyboards/ocean/wang_v2/readme.md:3]: Please rehost this on imgur.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/ocean/wang_v2/readme.md (hunk 2 above)
Comment (oceeean) [reply keyboards/ocean/wang_v2/readme.md]: changed to imgur

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Comment by oceeean** [comment]
thank you ❤️

**Pull request merged**